<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="894" started-at="2021-03-02T11:18:36 IST" finished-at="2021-03-02T11:18:37 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="894" started-at="2021-03-02T11:18:36 IST" finished-at="2021-03-02T11:18:37 IST">
      <class name="com.OrangeHRM.Sikuli_Integration_Example">
        <test-method status="FAIL" signature="FileUpload_UsingSikuli()[pri:0, instance:com.OrangeHRM.Sikuli_Integration_Example@536aaa8d]" name="FileUpload_UsingSikuli" duration-ms="859" started-at="2021-03-02T11:18:36 IST" finished-at="2021-03-02T11:18:36 IST">
          <exception class="java.lang.ExceptionInInitializerError">
            <full-stacktrace>
              <![CDATA[java.lang.ExceptionInInitializerError
at com.OrangeHRM.Sikuli_Integration_Example.FileUpload_UsingSikuli(Sikuli_Integration_Example.java:19)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException
at org.sikuli.script.RunTime.init(RunTime.java:549)
at org.sikuli.script.RunTime.get(RunTime.java:291)
at org.sikuli.script.RunTime.get(RunTime.java:126)
at org.sikuli.script.RunTime.get(RunTime.java:316)
at org.sikuli.script.Region.<clinit>(Region.java:31)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ExceptionInInitializerError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FileUpload_UsingSikuli -->
      </class> <!-- com.OrangeHRM.Sikuli_Integration_Example -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
